lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h

```diff
@@ -655,6 +655,34 @@ class ConvertTritonGPUOpToLLVMPatternBase {
     llvm_unreachable("unsupported emitOffsetForLayout");
   }
 
+#ifdef USE_ROCM
+  void emitMfmaOffsetForCTA(const MfmaEncodingAttr &mfmaLayout,
+                            SmallVector<SmallVector<unsigned>> &offsets,
+                            unsigned ctaOffsetX, unsigned ctaOffsetY) const {
+    // MFMA output tile consists of repeated "dot operand B" layout blocks along
+    // row axis. This variable defines number of these blocks
+    const unsigned numBlocks = 4;
```

Comment:
Again, note for future us: the hard coded 4 only works for 32x32 mfma instructions

The `block` is used everywhere already for various things. Can you pick another name for it. Maybe group?